test/integrationtests/voight_kampff/test_setup.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import argparse
+from argparse import RawTextHelpFormatter
+from glob import glob
+from os.path import join, dirname, expanduser, basename
+from pathlib import Path
+from random import shuffle
+import shutil
+import sys
+
+import yaml
+
+from msm import MycroftSkillsManager
+from msm.exceptions import MsmException
+
+from .generate_feature import generate_feature
+
+"""Test environment setup for voigt kampff test
+
+The script sets up the selected tests in the feature directory so they can
+be found and executed by the behave framework.
+
+The script also ensures that the skills marked for testing are installed and
+that anyi specified extra skills also gets installed into the environment.
+"""
+
+FEATURE_DIR = join(dirname(__file__), 'features') + '/'
+
+
+def copy_feature_files(source, destination):
+    """Copy all feature files from source to destination."""
+    # Copy feature files to the feature directory
+    for f in glob(join(source, '*.feature')):
+        shutil.copyfile(f, join(destination, basename(f)))
+
+
+def copy_step_files(source, destination):
+    """Copy all python files from source to destination."""
+    # Copy feature files to the feature directory
+    for f in glob(join(source, '*.py')):
```

Comment:
more meaningful name

Reply:
In this case I prefer to keep it a single letter since
- it's an iterator variable
- it's inside a two line function with descriptive name
- it makes the code easier to read